espnet2/train/trainer.py

```diff
@@ -512,11 +520,6 @@ def train_one_epoch(
         ):
             assert isinstance(batch, dict), type(batch)
 
-            if distributed:
```

Comment:
I performed several experiments (ASR, SSL) and found that this was useless. But we may need to experiment with other batch samplers to make sure